website/client/components/userMenu/profile.vue

```diff
@@ -5,16 +5,15 @@ div
       .profile-actions
         button.btn.btn-secondary(@click='sendMessage()')
           .svg-icon.message-icon(v-html="icons.message")
-        button.btn.btn-secondary(v-if='user._id !== this.userLoggedIn._id && userLoggedIn.inbox.blocks.indexOf(user._id) === -1', :tooltip="$t('unblock')",
+        button.btn.btn-secondary(@click='openSendGemsModal()')
+          .svg-icon.gift-icon(v-html="icons.gift")
+        button.btn.btn-secondary(v-if='user._id !== this.userLoggedIn._id && userLoggedIn.inbox.blocks.indexOf(user._id) === -1',
+          :tooltip="$t('unblock')",
```

Comment:
it seems that `:tooltip` isn't working anymore, not your fault^^, its probably because of bootstrap-vue

could you change `:tooltip` to `:title` and  `tooltip-placement='right'` to `v-b-tooltip.hover.right=""` ? then this will work again

Reply:
Okay I fixed it, I did notice the gift and message tooltip aren't working as well (unless that was intentional). Did you want me to add it to them as well?

Reply:
sure, lets add some tooltips to the other buttons too :+1: